src/extensions/reorder-columns/bootstrap-table-reorder-columns.js

```diff
@@ -121,8 +121,16 @@ $.BootstrapTable = class extends $.BootstrapTable {
       dragaccept: this.options.dragaccept,
       clickDelay: 200,
       dragHandle: '.th-inner',
-      restoreState: order,
-      beforeStop: () => {
+      restoreState: order ? order : this.columnsSortOrder,
+      beforeStop: (table) => {
+        const sortOrder = {}
+        table.el.find('th').each((i, el) => {
+          sortOrder[$(el).data('field')] = i
+        })
+
+        this.columnsSortOrder = sortOrder
+        this.persistReorderColumnsState(this)
```

Comment:
What happen if cookie is not used? Throw an exception?

Reply:
Nothing happens :-)

Reply:
ok, cool